docs/ja/user-guide/configuration/workflow.md

```diff
@@ -234,6 +236,54 @@ jobs:
 
 サンプルリポジトリ: https://github.com/screwdriver-cd-test/workflow-remote-requires-example
 
+## リモートジョイン
+リモートジョインジョブを作成することもできます。この機能がサポートされているかは、クラスタ管理者に確認してください。
+
+#### 例
+以下の例では、このパイプライン3は、パイプライン2の internal_fork、external_fork とパイプライン4の external_fork が成功した場合に `join_job` を開始します。
```

Comment:
`internal_fork` is not under pipeline2

Reply:
`internal_fork` is under pipeline3. FIxed.